File: snapshots/output-net10.0/syntax/Main/Properties.cs
  using System.Diagnostics.CodeAnalysis;
//      ^^^^^^ read_access scip-dotnet nuget . . System/
//      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ import scip-dotnet nuget . . CodeAnalysis/
//             ^^^^^^^^^^^ read_access scip-dotnet nuget . . Diagnostics/
//                         ^^^^^^^^^^^^ read_access scip-dotnet nuget . . CodeAnalysis/

  namespace Main;
//          ^^^^ read_access scip-dotnet nuget . . Main/
//               enclosing_range 2:0-18:1

  [SuppressMessage("ReSharper", "all")]
// ^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
//                 enclosing_range 4:0-18:1
  public class Properties
//             ^^^^^^^^^^ definition scip-dotnet nuget . . Main/Properties#
//                        enclosing_range 2:0-18:1
//                        documentation ```cs\nclass Properties\n```
//                        kind Class
  {
      byte Get { get; }
//         ^^^ definition scip-dotnet nuget . . Main/Properties#Get.
//             enclosing_range 4:0-18:1
//             documentation ```cs\nprivate byte Properties.Get { get; }\n```
//             kind Property
//             relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Byte#

      char Set
//         ^^^ definition scip-dotnet nuget . . Main/Properties#Set.
//             enclosing_range 4:0-18:1
//             documentation ```cs\nprivate char Properties.Set { set; }\n```
//             kind Property
//             relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Char#
      {
          set { throw new NotImplementedException(); }
//                        ^^^^^^^^^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 System/NotImplementedException#
//                                                enclosing_range 9:4-12:5
      }

      uint GetSet { get; set; }
//         ^^^^^^ definition scip-dotnet nuget . . Main/Properties#GetSet.
//                enclosing_range 4:0-18:1
//                documentation ```cs\nprivate uint Properties.GetSet { get; set; }\n```
//                kind Property
//                relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/UInt32#
      long SetGet { set; get; }
//         ^^^^^^ definition scip-dotnet nuget . . Main/Properties#SetGet.
//                enclosing_range 4:0-18:1
//                documentation ```cs\nprivate long Properties.SetGet { get; set; }\n```
//                kind Property
//                relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int64#

      string? Init { get; init; }
//            ^^^^ definition scip-dotnet nuget . . Main/Properties#Init.
//                 enclosing_range 4:0-18:1
//                 documentation ```cs\nprivate string? Properties.Init { get; init; }\n```
//                 kind Property
//                 relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/String#
  }

File: snapshots/output-net10.0/syntax/VBMain/Properties.vb
  Imports System.Diagnostics.CodeAnalysis
'         ^^^^^^ read_access scip-dotnet nuget . . System/
'                enclosing_range 0:8-0:26
'         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ import scip-dotnet nuget . . CodeAnalysis/
'                                         enclosing_range 0:8-0:39
'                ^^^^^^^^^^^ read_access scip-dotnet nuget . . Diagnostics/
'                            enclosing_range 0:8-0:26
'                            ^^^^^^^^^^^^ read_access scip-dotnet nuget . . CodeAnalysis/
'                                         enclosing_range 0:8-0:39

  Namespace VBMain
'           ^^^^^^ read_access scip-dotnet nuget . . VBMain/
'                  enclosing_range 2:10-2:16
      <SuppressMessage("ReSharper", "all")>
'      ^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
'                      enclosing_range 3:5-3:20
      Public Class Properties
'                  ^^^^^^^^^^ definition scip-dotnet nuget . . VBMain/Properties#
'                             enclosing_range 3:4-4:27
'                             documentation ```vb\nClass Properties\n```
'                             kind Class
          Private ReadOnly Property [Get] As Byte
'                                   ^^^^^ definition scip-dotnet nuget . . VBMain/Properties#Get.
'                                         enclosing_range 5:8-5:47
'                                         documentation ```vb\nPrivate ReadOnly Property Properties.Get As Byte\n```
'                                         kind Property
'                                         relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Byte#

          Private WriteOnly Property [Set] As Char
'                                    ^^^^^ definition scip-dotnet nuget . . VBMain/Properties#Set.
'                                          enclosing_range 7:8-7:48
'                                          documentation ```vb\nPrivate WriteOnly Property Properties.Set As Char\n```
'                                          kind Property
'                                          relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Char#
              Set(ByVal value As Char)
'                       ^^^^^ definition scip-dotnet nuget . . VBMain/Properties#set_Set().(value)
'                             enclosing_range 8:12-8:36
'                             documentation ```vb\nvalue As Char\n```
'                             kind Parameter
'                             relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Char#
                  Throw New NotImplementedException()
'                           ^^^^^^^^^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 System/NotImplementedException#
'                                                   enclosing_range 9:22-9:51
              End Set
          End Property

          Private Property GetSet As UInteger
'                          ^^^^^^ definition scip-dotnet nuget . . VBMain/Properties#GetSet.
'                                 enclosing_range 13:8-13:43
'                                 documentation ```vb\nPrivate Property Properties.GetSet As UInteger\n```
'                                 kind Property
'                                 relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/UInt32#
          Private Property SetGet As Long
'                          ^^^^^^ definition scip-dotnet nuget . . VBMain/Properties#SetGet.
'                                 enclosing_range 14:8-14:39
'                                 documentation ```vb\nPrivate Property Properties.SetGet As Long\n```
'                                 kind Property
'                                 relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int64#
      End Class
  End Namespace
